src/styles/styles.css

```diff
@@ -21,14 +22,32 @@ body {
 .mobile-nav {
   display: block;
 }
+.mobile-nav-button {
+  border: none;
+  background: rgba(255, 255, 255, 0.7);
+  margin: 10px;
+  padding: 10px;
+}
 
 .profile-wrapper {
+  background: rgba(255, 255, 255, 0.6);
   display: none;
+  overflow: auto;
+  padding: 10px;
 }
 .profile-wrapper.show {
   display: block;
 }
 
+.player-list__header {
+  display: flex;
+}
+.player-list__content {
+  height: 70%;
```

Comment:
this should't be fixed number and it should have scroll if the inner content is too long
![image](https://user-images.githubusercontent.com/980955/110463855-b5695a00-80d2-11eb-8ad5-dc6090458705.png)

Reply:
I'm unable to set all height and overflow settings by myself so I have created a separate issue for this: #34